contracts/evoting/evoting.go

```diff
@@ -825,14 +854,58 @@ func (e evotingCommand) manageAdminForm(snap store.Snapshot, step execution.Step
 		return xerrors.Errorf("failed to marshal Form : %v", err)
 	}
 
-	err = snap.Set(formID, formBuf)
+	err = snap.Set(formIDBuf, formBuf)
 	if err != nil {
 		return xerrors.Errorf("failed to set value: %v", err)
 	}
 
 	return nil
 }
 
+// isAdmin Check whether a user is in an Admin List
+func (e evotingCommand) isAdmin(form types.AdminList, txPerformingUser string) (bool, error) {
+	// If it found the AdminList
+	// Check that the performing user is Admin
+	performingUserPerm, err := form.GetAdminIndex(txPerformingUser)
+	if err != nil {
+		return false, xerrors.Errorf("couldn't retrieve admin permission of the performing user: %v", err)
+	}
+
+	if performingUserPerm < 0 {
+		return false, xerrors.Errorf("the performing user %v doesn't have the permission to add admin", txPerformingUser)
+	}
```

Comment:
the function is behaving correctly (returning false if the admin cannot be found), so there's no error